### ISSUE TITLE
'new NativeEventEmitter()' was called with a non-null argument

### ISSUE THREAD

**Issue by kris1803** [pushy/pushy-react-native#77]
When importing library to project, doing correctly all steps and building app, two warnings appear : 

![image](https://user-images.githubusercontent.com/78277770/181441262-b6c9e5f2-2aa6-426f-84d4-f8afb010f06a.png)

`new NativeEventEmitter()` was called with a non-null argument without the required `addListener` method.
`new NativeEventEmitter()` was called with a non-null argument without the required `removeListeners` method.

It is logged in console and appears as warning on device screen. I'm using expo, bare workflow.
I found a solution:
adding to Pushy.js two lines of code : 
```ts
Pushy.addListener = () => {}
Pushy.removeListeners = () => {}
``` 

Then no more warnings. This is an issue from 2021, of how react native changed.

**Comment by pushy** [comment]
Hi @kris1803,
Thanks for your report and for the suggested workaround.

Are you able to get our SDK working properly in Expo? Last we checked, [HeadlessJS tasks were broken](https://forums.expo.dev/t/appregistry-registerheadlesstask/15425) in Expo. Please confirm background notification delivery works on Android with Expo and we will resolve these warnings.

**Comment by kris1803** [comment]
Yes, headless tasks work (or if you talk about pushy register and listen more precisely), so app can wake up when it was killed to receive notification. So it works properly, but meanwhile I didn't test subscriptions to notifications.

**Comment by pushy** [comment]
Hi @kris1803,
Thanks for confirming. 

This warning has now been resolved in https://github.com/pushy/pushy-react-native/commit/a1a1fe0ca8e9ca56755b651884a2f461f9a08e0f and published to `npm` in version `1.0.23`. 🎉
